lib/reporters/mini.js

```diff
@@ -10,6 +11,10 @@ function MiniReporter() {
 		return new MiniReporter();
 	}
 
+	var spinnerDef = spinners.dots;
+	this.spinnerFrames = spinnerDef.frames;
+	this.spinnerInterval = spinnerDef.interval;
```

Comment:
Also, this way leaves us open to having a command line option to configure your spinner:

```
$ ava --spinner=balloon
```

then

```js
var spinnerDef = spinners[config.spinner];
```